balena/models/release.py

```diff
@@ -81,6 +106,24 @@ def get_all_by_application(self, app_id):
 
         return self.__get_by_option('belongs_to__application', app_id)
 
+    def get_latest_by_application(self, app_id):
+        """
+        Get the latest successful release for an application.
+
+        Args:
+            app_id (str): applicaiton id.
+
+        Returns:
+            dict: release info.
+
+        """
+
+        raw_query = "$top=1&$filter=belongs_to__application%20eq%20'{}'%20and%20status%20eq%20'success'".format(app_id)
```

Comment:
What is `belongs_to__application eq '{}'`?

Reply:
it's a formatted string resolves to `belongs_to__application eq 'app_id'`. I've not had time to implement something the same as pineJS in Python so we're using raw OData query for some complex query in the Python SDK.